### ISSUE TITLE
Adding Members to a new project - No users available

### ISSUE THREAD

**Issue by peteingithub** [rancher/dashboard#4004]
2.6.0-rc6

![image](https://user-images.githubusercontent.com/86831399/130734047-8f97c050-72f7-463a-9a7d-dbfe2dc3ca6b.png)

**Comment by gaktive** [comment]
Once you start typing, member names start propagating. However, it doesn't look pretty.

**Comment by peteingithub** [comment]
I thought I tried that, but I guess not. Seems to work, but it is a bit awkward. Filtering by typing would be cool, but all users should be listed in the dropdown. 
Typing will not fill in external users, right?

**Comment by peteingithub** [comment]
External users seem to also work, but you need to start typing the name of the ID.

![image](https://user-images.githubusercontent.com/86831399/130828767-c8817344-58ad-4c5b-90ee-f5273ebbe2ae.png)

**Comment by nwmac** [comment]
@jordojordo Talk to @richard-cox on this one - @rak-phillip is doing something related. This could be an issue with the local users that needs fixing.

**Comment by slickwarren** [comment]
Testing on v2.6-head (1f5852d):
I'm not sure what the intended behavior for each case is from this ticket, but here's what I observe for adding users to projects with `Github` as the external source of users:


* when user clicks `add user`, a menu appears with no users listed by default -- I think this is what is expected, pass
<img width="722" alt="Screen Shot 2021-09-18 at 12 49 01 PM" src="https://user-images.githubusercontent.com/16691014/133906967-567a7360-88d3-4a36-9b37-5783e671b5e2.png">

* **when user clicks the dropdown in the menu, groups appear, but no users -- Reopening for this, I'm not sure if this is the intended behavior. Also, a new group that was not there by default does not show up in this list.** 
<img width="450" alt="Screen Shot 2021-09-18 at 12 46 32 PM" src="https://user-images.githubusercontent.com/16691014/133906971-aa616266-8ead-469b-aa2f-a51fb8ccf177.png">

* when user begins to type, filtered users are populated in the dropdown
  * Local users are showing up in the search -- pass
  * <img width="232" alt="Screen Shot 2021-09-18 at 12 56 28 PM" src="https://user-images.githubusercontent.com/16691014/133907071-0ce6b3d3-88c9-44a1-9db6-b095ac7b2582.png">

**Comment by richard-cox** [comment]
@slickwarren It all comes down to what's in the `/v3/principals` response from the backend. For scenarios where a non-local auth provider, like Github, is configured the user may have visibility on a LOT of users so none are shown. I think if that's an issue it needs to be handled separately in a `rancher/rancher` specific issue

**Comment by izaac** [comment]
Rancher version `v2.6-head (09/22/2021)` _053d4c4_
Auth Provider: ` Azure AD`
I was able to see the populated users and groups. Adding a new group also appeared typing in the autocomplete input but after refreshing the groups memberships (`Users & Authentication` -> `Refresh Group Membership`).

But there's a small issue I've noticed in the UI when I open the Add Project Member. The text displayed in the control input says "**Search for a member to provider cluster access**" but im actually at the project level access.

After fixing this I think we're good to close.

![Screen Shot 2021-09-22 at 11 52 27 AM](https://user-images.githubusercontent.com/27659/134404776-34476e5a-25b5-40df-a766-cc707f1e11c4.png)
